IMAGES.rst

```diff
@@ -247,29 +250,92 @@ significant changes are done in the Dockerfile.CI.
 The images are named differently (in Docker definition of image names - registry URL is part of the
 image name if DockerHub is not used as registry). Also GitHub has its own structure for registries
 each project has its own registry naming convention that should be followed. The name of
-images for GitHub registry are:
+images for GitHub registry are different as they must follow limitation of the registry used.
+
+We are still using Github Packages as registry, but we are in the process of testing and switching
+to GitHub Container Registry, and the naming conventions are slightly different (GitHub Packages
+required all packages to have "organization/repository/" URL prefix ("apache/airflow/",
+where in GitHub Container Registry, all images are in "organization" not in "repository" and they are all
+in organization wide "apache/" namespace rather than in "apache/airflow/" one).
+We are adding "airflow-" as prefix for image names of all Airflow images instead.
+The images are linked to the repository via ``org.opencontainers.image.source`` label in the image.
+
+Naming convention for GitHub Packages
+-------------------------------------
+
+Images built as "Run ID snapshot":
+
+.. code-block:: bash
+
+  ghcr.io/apache/airflow/<BRANCH>-pythonX.Y-ci-gcr-v1:<RUNID>    - for CI images
+  ghcr.io/apache/airflow/<BRANCH>-pythonX.Y-gcr-v1:<RUNID>       - for production images
+  ghcr.io/apache/airflow/<BRANCH>-pythonX.Y-build-gcr-v1:<RUNID> - for production build stage
+  ghcr.io/apache/airflow/pythonX.Y-<BRANCH>-gcr-v1:X.Y-slim-buster-<RUN_ID>  - for base python images
+
+Latest images (pushed when master merge succeeds):
 
 .. code-block:: bash
 
-  docker.pkg.github.com/apache/airflow/<BRANCH>-pythonX.Y       - for production images
-  docker.pkg.github.com/apache/airflow/<BRANCH>-pythonX.Y-ci    - for CI images
-  docker.pkg.github.com/apache/airflow/<BRANCH>-pythonX.Y-build - for production build state
-  docker.pkg.github.com/apache/airflow/pythonX.Y-<BRANCH>       - for base python images
+  ghcr.io/apache/airflow/<BRANCH>-pythonX.Y-ci-gcr-v1:latest    - for CI images
```

Comment:
We shouldn't need the `gcr` 5n the image name too.

Reply:
I also don't see in the code where the `gcr` part of the image name comes from -- are the docs incorrect now?

Reply:
Yeah. It was documenting "future" change coming from a follow-up PR I had about switching.
But yeah "-gcr" can be remved. And since  we are at -v2 now, I iwll leave it with "-v2".